apps/api/v1/lib/utils/retrieveScopedAccessibleUsers.ts

```diff
@@ -0,0 +1,89 @@
+import prisma from "@calcom/prisma";
```

Comment:
Do we not have these functions already written somewhere in the core libraries of the code base that we could reuse?

Reply:
Let me look into it. These functions evolved in the past few commits so it is possible this they're already in the repo in the current state. 🙏